deploy/cloudwash_job.yaml

```diff
@@ -73,3 +80,6 @@ parameters:
   description: "Schedule for cron job"
   value: ''
   required: true
+- name: CLOUDWASH_ARGS
+  required: true
+  value: '["aws", "azure", "--nics", "--vms", "--discs", "--images"]'
```

Comment:
We don't want to delete images.

Reply:
`'["aws", "azure", "--nics", "--vms", "--discs", "--images"]' ` is not a valid command. 
Can you confirm it?

Reply:
Thanks for the review!
I would like to get some feedback from @mshriver too.
and yes those are parameterized lists for both the cloud resources.
that's why it's like combined params but yeah let's discuss it when mike is back.

Reply:
looks like cloudwash will only support one cloud at a time.
and as per akhil mentioned i am creating a new container for other resource 
Thanks !

Reply:
ref : https://redhat-internal.slack.com/archives/C04B2FX09FV/p1705412248915979?thread_ts=1705355987.988969&cid=C04B2FX09FV

Reply:
Please don't post the internal links on github